doc/source/theory/FlavorSpace.rst

```diff
@@ -117,11 +117,11 @@ In this case the QED :math:`\otimes` QCD evolution basis that performs the maxim
   \Sigma_u & = u^+ + c^+ + t^+ \\
   \Sigma_d & = d^+ + s^+ + b^+ \\
   \Sigma &= \Sigma_u + \Sigma_d \\
-  \Delta_\Sigma &= \Sigma_u - \Sigma_d \\
+  T_0 &= \Sigma_u - \Sigma_d \\
   V_u & = u^- + c^- + t^- \\
   V_d & = d^- + s^- + b^- \\
   V &= V_u + V_d \\
-  \Delta_V &= V_u - V_d \\
+  V_0 &= V_u - V_d \\
   T_1^u &=u^+ - c^+ \\
```

Comment:
It's not clear that `\Sigma_u` and its relatives are spelled out to be used inside the other combinations, but they are not part of the basis.

I'd suggest to provide them in a separate `math::` block before, and then spell in a second block the all and only the members of the actual basis.

Reply:
Fine that there is the `span` at the end, but this way would be much clearer.